addons/repos/main.yaml

```diff
@@ -1,5 +1,8 @@
 ---
 packages:
+  - name: fluxcd-source-controller
+    versions:
+      - 0.21.2
```

Comment:
@EstebanFS Can you alphabetize this? Doing that will also clear the merge conflict.

Reply:
I just putted fluxcd-source-controller in the right spot!